### DISCUSSION TITLE
Error: Incorrect padding

### DISCUSSION THREAD

**Discussion by Tiaranso** [ashleykleynhans/runpod-worker-real-esrgan#4] [Q&A]
{
  "delayTime": 5436,
  "error": "{\"error_type\": \"<class 'binascii.Error'>\", \"error_message\": \"Incorrect padding\", \"error_traceback\": \"Traceback (most recent call last):\\n  File \\\"/usr/local/lib/python3.10/dist-packages/runpod/serverless/modules/rp_job.py\\\", line 133, in run_job\\n    handler_return = handler(job)\\n  File \\\"/workspace/runpod-worker-real-esrgan/rp_handler.py\\\", line 244, in handler\\n    return upscaling_api(validated_input['validated_input'])\\n  File \\\"/workspace/runpod-worker-real-esrgan/rp_handler.py\\\", line 197, in upscaling_api\\n    source_image = base64.b64decode(source_image_data)\\n  File \\\"/usr/lib/python3.10/base64.py\\\", line 87, in b64decode\\n    return binascii.a2b_base64(s)\\nbinascii.Error: Incorrect padding\\n\", \"hostname\": \"kll3hswicv5u00-64411545\", \"worker_id\": \"kll3hswicv5u00\", \"runpod_version\": \"1.3.7\"}",
  "executionTime": 1051,
  "id": "cb800422-0754-4cb0-81fb-3cb34d14f6bd-u1",
  "status": "FAILED"
}

{
  "delayTime": 389,
  "error": "{\"error_type\": \"<class 'binascii.Error'>\", \"error_message\": \"Incorrect padding\", \"error_traceback\": \"Traceback (most recent call last):\\n  File \\\"/usr/local/lib/python3.10/dist-packages/runpod/serverless/modules/rp_job.py\\\", line 133, in run_job\\n    handler_return = handler(job)\\n  File \\\"/workspace/runpod-worker-real-esrgan/rp_handler.py\\\", line 244, in handler\\n    return upscaling_api(validated_input['validated_input'])\\n  File \\\"/workspace/runpod-worker-real-esrgan/rp_handler.py\\\", line 197, in upscaling_api\\n    source_image = base64.b64decode(source_image_data)\\n  File \\\"/usr/lib/python3.10/base64.py\\\", line 87, in b64decode\\n    return binascii.a2b_base64(s)\\nbinascii.Error: Incorrect padding\\n\", \"hostname\": \"kll3hswicv5u00-64411545\", \"worker_id\": \"kll3hswicv5u00\", \"runpod_version\": \"1.3.7\"}",
  "executionTime": 1043,
  "id": "a99dc9b8-c9d0-4e82-8b1e-9b0daa4942d4-u1",
  "status": "FAILED"
}

How to solve this problem?

**Comment by ashleykleynhans** [comment]
Base64 encoding issue as stated in the error message, base64 encode the image data correctly.
